apps/kol-amelamdim/pages/weekly-article/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { StyledPageContainer } from '@kol-amelamdim/styled';
+import { Typography, styled } from '@mui/material';
+import axios from '../../api';
+
+const WeeklyArticleContainer = styled(StyledPageContainer)`
+  padding: 125px 0 70px;
+  h1,
+  h2,
+  p {
+    margin: 0;
+  }
+`;
+
+const Index = ({ activeArticle }) => {
+  return (
+    <WeeklyArticleContainer>
+      <Typography variant="h1" component="h1">
+        {activeArticle.title}
+      </Typography>
+      <Typography variant="h2" component="h2">
+        {activeArticle.description}
+      </Typography>
+      <div
+        className="content"
+        dangerouslySetInnerHTML={{ __html: activeArticle.content }}
+      ></div>
+    </WeeklyArticleContainer>
+  );
+};
+
+export async function getServerSideProps() {
+  try {
+    const activeArticle = await axios.get('/api/get-active-weekly-article');
+    return {
+      props: { activeArticle: activeArticle.data },
+    };
+  } catch (e) {
+    return { props: {} };
```

Comment:
i think we should inform the user in case of something went wrong..

Reply:
fixed

Reply:
<img width="1504" alt="image" src="https://user-images.githubusercontent.com/110565257/189120915-457d4e14-b81f-41f0-9ad7-9ab127a2c8f1.png">